js/ui/html_editor/modules/dropImage.js

```diff
@@ -6,16 +6,16 @@ import { each } from '../../../core/utils/iterator';
 import browser from '../../../core/utils/browser';
 import { getWindow } from '../../../core/utils/window';
 
-let DropImageModule = {};
+import EmptyModule from './empty';
+import BaseModule from './base';
 
-if(Quill) {
-    const BaseModule = Quill.import('core/module');
+let DropImageModule = EmptyModule;
```

Comment:
```suggestion
let DropImageModule = BaseModule;
```
If `!Quill` => `BaseModule === EmptyModule`

And so on for other modules